swift/driver_swiftui/Tests/UnitTests/ProviderServiceTests.swift

```diff
@@ -0,0 +1,192 @@
+/*
```

Comment:
Just double checking, we want to also publish tests?

Reply:
Thanks. Yes, this is fine.